crates/core/src/common.rs

```diff
@@ -0,0 +1,218 @@
+use kitsune2_api::{
+    AgentInfoSigned, DhtArc, DynLocalAgentStore, DynPeerStore, K2Result,
+};
+use std::collections::HashSet;
+use std::sync::Arc;
+
+/// Get all remote agents from the peer store.
+///
+/// Uses the [DynPeerStore] to get all agents and the [DynLocalAgentStore] to filter out local
+/// agents.
+pub async fn get_all_remote_agents(
```

Comment:
This one is used by K2 core, the others are only used by Holochain in my local patched Holochain version